### PR TITLE
Feature/signup screen

### PR DESCRIPTION
**Pull request by alandours**
- Add name and gender fields to the signup form
- Add validations to the login and signup forms
- Add Picker component

Screenshots:

### CODE REVIEW

File: src/components/common/Picker/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        }}
+        useNativeAndroidPickerStyle={false}
+        items={items}
+        Icon={() => null}
+        {...props}
+      />
+      {touched && !!error && <Text accessibilityLabel="form-error">{error}</Text>}
+    </View>
+  );
+};
```
Comment (Stukz) [inline src/components/common/Picker/index.js:24]:
```suggestion
const Picker = ({ label, items, error, active, touched, testID, ...props }) => (
    <View style={styles.pickerContainer}>
      {label && <Text style={styles.label}>{label}</Text>}
      <RNPickerModule
        textInputProps={{
          style: [styles.picker, touched && !!error && styles.pickerError],
        }}
        useNativeAndroidPickerStyle={false}
        items={items}
        Icon={() => null}
        {...props}
      />
      {touched && !!error && <Text accessibilityLabel="form-error">{error}</Text>}
    </View>
  );
```

File: src/components/common/Picker/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Picker.propTypes = {
+  label: string,
+  items: arrayOf(shape({ label: string, value: string })),
```
Comment (Stukz) [inline src/components/common/Picker/index.js:28]: I just want to know if you don't send the items you don't get an error message?

File: src/validations/loginValidations.js [hunk 3]
```diff
@@ -1,9 +1,17 @@
+import strings from 'localization';
+
+const { VALIDATIONS } = strings;
+
 export default {
   email: {
     presence: true,
-    email: true,
+    email: { message: VALIDATIONS.email.email },
```
Comment (Stukz) [inline src/validations/loginValidations.js:8]: I got the idea here but It's not that clear maybe you can rename it to
```suggestion
    email: { message: VALIDATIONS.email.format },
```

File: src/validations/signUpValidations.js [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
     presence: true,
+    length: {
+      minimum: 8,
+      message: VALIDATIONS.password.length,
+    },
   },
   passwordConfirmation: {
     presence: true,
-    equality: { attribute: 'password' },
+    equality: { attribute: 'password', message: VALIDATIONS.passwordConfirmation.equality },
```
Comment (Stukz) [inline src/validations/signUpValidations.js:22]: maybe as a readability improvement you can do 
```suggestion
    equality: { 
       attribute: 'password', 
       message: VALIDATIONS.passwordConfirmation.equality
   },
```

**Review by Stukz — APPROVED** [review]

**Comment by Stukz** [comment]
I forgot about the test cases, take a look at that because the CI Test is failing

File: src/components/SignUpForm/index.js [hunk 5]
```diff
@@ -53,25 +56,44 @@ const SignUpForm = ({ onSubmit }) => {
...
         testID="password-input"
+        placeholder={strings.SIGN_UP.passwordPlaceholder}
+        disabled={status === LOADING}
         {...inputProps(FIELDS.password)}
       />
       <Input
         label={strings.SIGN_UP.passwordConfirmation}
         secureTextEntry
         testID="confirm-password-input"
+        disabled={status === LOADING}
```
Comment (tamarasuarezrod) [inline src/components/SignUpForm/index.js:85]: maybe you can define this as a constant `const isLoading = status === LOADING` so you can reuse it on these lines

**Review by tamarasuarezrod — APPROVED** [review]
👏 👏 well done! very consistent

**Pull request merged**